docker/Dockerfile

```diff
@@ -88,13 +113,57 @@ RUN cp /FV3/conf/$configure_file \
     if [ ! -z $compile_option ]; then sed -i "33i $compile_option" \
         /FV3/conf/configure.fv3; fi
 
+COPY FV3/atmos_cubed_sphere /FV3/atmos_cubed_sphere
+
+## Convert FV3GFS sources for serialize image
+##---------------------------------------------------------------------------------
+FROM fv3gfs-environment-serialbox AS fv3gfs-sources-true
+
+ENV FV3_BASE=/FV3/original
+COPY --from=fv3gfs-sources-false /FV3 $FV3_BASE
+COPY --from=fv3gfs-sources-false /stochastic_physics $(FV3_BASE)/../stochastic_physics 
+
+ARG compile_option
+ARG configure_file=configure.fv3.gnu_docker_serialize
+
+# copy appropriate configuration file to configure.fv3
+RUN cp $FV3_BASE/conf/$configure_file \
+        $FV3_BASE/conf/configure.fv3 && \
+    if [ ! -z $compile_option ]; then sed -i "33i $compile_option" \
+        $FV3_BASE/conf/configure.fv3; fi
+
+ENV SERIALBOX_OUTDIR=/FV3
+
+# build model with serialization
+RUN cd $FV3_BASE && make serialize_preprocess
+
+FROM fv3gfs-sources-$serialize AS fv3gfs-sources
```

Comment:
I don't see buildkit stuff being used, have I missed it?  I was trying to avoid having to build everything to get to the default version of the model vs the serialized version, but for tests since you want it all perhaps that's why it is not used? But the ML team can just build the non-serialized version, when they do so will both the fv3gfs-sources-true and fv3-gfs-sources-false get built?

Reply:
To use buildkit all you do is set `DOCKER_BUILDKIT=1` as an environment variable. It's been working well for me.

Reply:
but we aren't using that in the Makefile here? do you intend for that to be just used by the repos calling this one?

Reply:
The result should be the same whether you use buildkit or not. I haven't put it in any Makefile, each user should be able to select if they want to use buildkit. I just added it to my `~/.bash_profile`.

Reply:
hmm, should we mention it in the readme then (or maybe the readme's of the repos that submodule this)? It's gotta be soooo slow to compile the model twice and install Serialbox. If we start doing more docker pulls maybe it won't matter.

Reply:
I've added a short paragraph to the README.